app/views/users/circuitverse/edit.html.erb

```diff
@@ -126,4 +126,11 @@
     const selection = feedback.selection.value[feedback.selection.key];
     autoCompleteJS.input.value = selection;
   });
+  $('#remove_pic').change(function() {
+    if($(this).is(":checked")) {
+      $('#imgedit').attr('src', '/assets/images/thumb/Default.jpg');
```

Comment:
The image can't be directly using this path, you need to use `asset_url` or `image_path`

```suggestion
      $('#imgedit').attr('src', '<%= image_path('thumb/Default.jpg') %>');
```
Except this , the function seems to be working as expected !